tests/sentry/tasks/integrations/github/test_pr_comment.py

```diff
@@ -81,6 +82,7 @@ def setUp(self):
         self.pr_key = 1
         self.commit_sha = 1
         self.fingerprint = 1
+        patch("sentry.integrations.github.client.get_jwt", return_value=b"jwt_token_1").start()
```

Comment:
what is the difference between doing this here vs as a decorator? is it equivalent to attaching the decorator to the class?

Reply:
With this, we don't have to have an unused argument in each function, also any subclasses would also have this method patched, vs for class decorator, you'd still need to decorate the subclass.